lib/model/query/forms.js

```diff
@@ -319,18 +306,21 @@ createVersion.audit.withResult = true;
 
 // TODO: we need to make more explicit what .def actually represents throughout.
 // for now, enforce an extra check here just in case.
-const publish = (form) => async ({ Forms, Datasets, FormAttachments }) => {
+const publish = (form, sameAsCreate = false) => async ({ Forms, Datasets, FormAttachments }) => {
```

Comment:
I don't love the `sameAsCreate` name. I was just trying to get at the situation where you call publish() and it's meant to be combined with the previous create operation. I don't know if this is actually important to preserve... but one case where it might come up is seeing `updatedAt = null` on a form that has been created+published in one go.

Reply:
I don't know that it's a big deal, but if it's not much code, it seems nice to preserve the previous behavior.

In terms of the name, `sameAsCreate` sounds OK to me. Other ideas: `newForm`, `isNewForm`, `afterCreate`, `concurrentWithCreate`.

Reply:
I like `concurrentWithCreate`